packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,106 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Small toolbar example:
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const smallButton = <Button size="small">Small button</Button>;
+
+<Toolbar size="small" rightContent={[smallButton, smallButton]} />
+```
+
+## Toolbar with title
+
+```js
+// TODO(scottgrant): Use wonder-blocks-icon and wonder-blocks-icon-button
+const dismissIcon =
+    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
+const closeButton = (
+    <svg role="img" width="24px" height="24px">
+        <path d={dismissIcon} />
+    </svg>
+);
+
+<Toolbar
+	leftContent={closeButton}
+	title="Counting with small numbers"
+/>
+```
+
+## Toolbar with left and right content
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const dismissIcon =
+    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
+const closeButton = (
+    <svg role="img" width="24px" height="24px">
+        <path d={dismissIcon} />
+    </svg>
+);
+
+const startExercise = <Button>Next Video</Button>;
+
+<Toolbar
+	leftContent={closeButton}
+	rightContent={startExercise}
+	title="Counting with small numbers"
+/>
+```
+
+## Toolbar with multiple elements on the right
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {LabelMedium} = require("@khanacademy/wonder-blocks-typography");
+
+const nextExercise = <Button>Next exercise</Button>;
+const questionCount = <LabelMedium>7 questions</LabelMedium>;
+const tryAgain = <Button kind="secondary">Try again</Button>;
+
+<Toolbar
+	rightContent={[questionCount, tryAgain, nextExercise]}
+/>
+```
+
+## Header overflow text
```

Comment:
Does this do the right thing on mobile?  I wasn't able to test it on the styleguidist page.

Reply:
The examples here don't restructure themselves based on the docs in Zeplin and there's some overflow with long words on the right side (`exercise` overflows), but the spacing works as expected. There's a note about assuming components that use `Toolbar` will use a media wrapper to specify if we should render differently based on size. Do you think that seems okay?